src/cbc_sdk/platform/legacy_alerts.py

```diff
@@ -342,7 +342,8 @@ def set_time_range(self, key, **kwargs):
         Returns:
             AlertSearchQuery: This instance.
         """
-        if key not in ["create_time", "first_event_time", "last_event_time", "last_update_time"]:
+        key = self._doc_class.REMAPPED_ALERTS_V6.get(key,key)
+        if key not in ["create_time", "first_event_time", "last_event_time", "last_update_time", "backend_update_timestamp"]:
```

Comment:
backend_timestamp is also valid as a conversion from create_time

Reply:
incorporated in CPAPI_4813_V6_Unit_Tests